ai-platform-unified/notebooks/official/custom/custom-tabular-bq-managed-dataset.ipynb

```diff
@@ -29,22 +29,21 @@
         "id": "title"
```

Comment:
<p>Perhaps emphasize that we do it during prediction as well:</p><p> </p><p>Later during prediction, use these summary statistics again to normalize the testing data.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/394/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p>Done.</p>